code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -580,7 +580,7 @@
 		to_chat(src, "<span class='danger'>You cannot charge in your current state!</span>")
 		return
 
-	last_special = world.time + 200
+	last_special = world.time + 10
```

Comment:
Is this meant to be only + 10? Isn't trample their strongest ability?

Reply:
As far I have been told, it was supossed to be used with the other powers, but I can increase the cooldown.

Reply:
That would not work well anyways, due to the fact that all human powers share 1 cooldown.